build_tools/benchmarks/comparisons/setup_desktop.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# Install Bazel. From https://www.tensorflow.org/install/source
+npm install -g @bazel/bazelisk
+
+# Create root dir.
+ROOT_DIR=/tmp/mobilebert_benchmarks
+mkdir ${ROOT_DIR}
+mkdir ${ROOT_DIR}/models
+mkdir ${ROOT_DIR}/models/tflite
+mkdir ${ROOT_DIR}/models/iree
+mkdir ${ROOT_DIR}/test_data
+mkdir ${ROOT_DIR}/output
+
+wget https://storage.googleapis.com/iree-model-artifacts/tflite_squad_test_data.zip -O /tmp/tflite_squad_test_data.zip
+unzip /tmp/tflite_squad_test_data.zip -d ${ROOT_DIR}/test_data/
+wget https://storage.googleapis.com/iree-model-artifacts/mobilebert_float_384_gpu.tflite -O ${ROOT_DIR}/models/tflite/mobilebert_float_384_gpu.tflite
+
+# Build IREE source.
+SOURCE_DIR=/tmp/github
+mkdir ${SOURCE_DIR}
+cd ${SOURCE_DIR}
+
+#git clone https://github.com/google/iree.git
+git clone https://github.com/mariecwhite/iree.git
+
+cd iree
+git checkout origin/comparisons
```

Comment:
Is it possible to build with the current checkout or even with the uncommitted changes? So we can compare the current change with TFLite instead of a fixed version.

Reply:
It's possible with some small changes in the script. It's expected use is to build and run from a fresh checkout but I see how this would be useful for debugging. Do you mind if we move this to V2?